### PR TITLE
added project filter component

### PR DESCRIPTION
**Pull request by AbdurrahmanSogbesan**

### CODE REVIEW

File: src/components/ProjectFilter/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import Label from "../Label";
+import "./ProjectFilter.css";
+
+function ProjectFilter({ text }) {
```
Comment (hmoskva) [inline src/components/ProjectFilter/index.js]: @AbdurrahmanSogbesan this wouldn't work this way. The filter is to work with several labels, but youve done it to work with just one. The props it'll take will be _filters_ and _selectedFilters_. Your filter prop will be an array in a shape like `[{text: 'All', onClick: a function that adds that current filter to the selected ones, className: if filter is in selected then 'projectFilterCLicked' else 'projectFilter'}...]`. So you will map the filters into Labels. Let me know if you dont understand

Comment (hmoskva) [reply src/components/ProjectFilter/index.js]: @AbdurrahmanSogbesan in addition, the state for selectedFilters and filters will be in the projects page itself. So yull just pass them into the projectFilters as props from there while using it. Like `<ProjectFilters filters={[your filters]} selectedFilters={your selected filters state} />`

Comment (AbdurrahmanSogbesan) [reply src/components/ProjectFilter/index.js]: > @AbdurrahmanSogbesan this wouldn't work this way. The filter is to work with several labels, but youve done it to work with just one. The props it'll take will be _filters_ and _selectedFilters_. Your filter prop will be an array in a shape like `[{text: 'All', onClick: a function that adds that current filter to the selected ones, className: if filter is in selected then 'projectFilterCLicked' else 'projectFilter'}...]`. So you will map the filters into Labels. Let me know if you dont understand

Im not sure I understand this

Comment (AbdurrahmanSogbesan) [reply src/components/ProjectFilter/index.js]: > @AbdurrahmanSogbesan in addition, the state for selectedFilters and filters will be in the projects page itself. So yull just pass them into the projectFilters as props from there while using it. Like `<ProjectFilters filters={[your filters]} selectedFilters={your selected filters state} />`

I haven't made the page yet so this is just for now

File: src/components/ProjectFilter/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import "./ProjectFilter.css";
+
+function ProjectFilter({filters}) {
+  const [selectedFilter, setSelectedFilter] = useState([]);
+  return (
+    <div>
+      {filters.map((filter) => (
+        <Label
+          text={filter}
+          onClick={() => setSelectedFilter(filter)}
```
Comment (hmoskva) [inline src/components/ProjectFilter/index.js:12]: @AbdurrahmanSogbesan the initial selectedfilter should be an empty string not an array

**Pull request merged**